bridges/otelzap/core.go

```diff
@@ -123,14 +134,58 @@ func (o *Core) Sync() error {
 	return nil
 }
 
-// TODO
 // Check determines whether the supplied Entry should be logged using core.Enabled method.
 func (o *Core) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.CheckedEntry {
+	if o.Enabled(ent.Level) {
+		return ce.AddCore(ent, o)
+	}
 	return ce
 }
 
 // TODO
 // Write method encodes zap fields to OTel logs and emits them.
 func (o *Core) Write(ent zapcore.Entry, fields []zapcore.Field) error {
+	r := log.Record{}
+	r.SetTimestamp(ent.Time)
+	r.SetBody(log.StringValue(ent.Message))
+	r.SetSeverity(getOTelLevel(ent.Level))
+
+	if len(fields) > 0 {
+		// TODO map zap field to otel attributes
+		fmt.Println("TODO")
+	} else {
+		r.AddAttributes(o.attr...)
+	}
```

Comment:
This can be  changed to multiple todo comments e.g.

```suggestion
	// TODO: Handle attributes passed via fields.
	// TODO: Handle attributes passed via With.
	// TODO: Handle context.Context containg trace context.
	// TODO: Handle zap.Namespace.
	// TODO: Handle logger name.
```